ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/TimeModel.scala

```diff
@@ -30,14 +31,13 @@ case class TimeModel private (
       ledgerTime: Instant,
       recordTime: Instant
   ): Either[String, Unit] = {
-    val lowerBound = recordTime.minus(minSkew)
-    val upperBound = recordTime.plus(maxSkew)
+    val lowerBound = minLedgerTime(recordTime, minSkew)
+    val upperBound = maxLedgerTime(recordTime, maxSkew)
     if (ledgerTime.isBefore(lowerBound) || ledgerTime.isAfter(upperBound))
```

Comment:
```suggestion
    val lowerBound = minRecordTime(ledgerTime, minSkew)
    val upperBound = maxRecordTime(ledgerTime, maxSkew)
    if (recordTime.isBefore(lowerBound) || recordTime.isAfter(upperBound))
```
If you replace the checks this way, then you don't need the `min/maxLedgerTime` methods at all and no need to document/test that minLedgerTime/maxRecordTime have a joint contract.

Reply:
Yes that was my first idea as well, my only problem with that is that I would have to change the message below and I'm not sure of the impacts; @miklos-da what's your take?

Reply:
Also @rautenrieth-da you might be able to comment on this.

Reply:
I don't think changing the rejection message should be a problem, i.e., it's not part of any contract (the fact that we are producing a `RejectionReason.InvalidLedgerTime` is).